assets/css/admin.scss

```diff
@@ -3903,7 +3913,7 @@ img.ui-datepicker-trigger {
 			margin: 0 !important;
 		}
 
-		h3.stats_range {
+		div.stats_range,h3.stats_range {
```

Comment:
Looks like not done up until now but technically each selector should be on its own line. I usually at least space them https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/#structure